recipe/meta.yaml

```diff
@@ -7,44 +7,51 @@ package:
 source:
   fn: impyla-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/i/impyla/impyla-{{ version }}.tar.gz
-  md5: 6202a37a16a98473b1c9ec9bbb9a4349
+  sha256: 3fb9a38725cd471833146633aaa9f8916d64035796e7b8dfc6f5a392bef5f184
 
 build:
-  number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  number: 1
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python
     - setuptools
-    - six
-    - bitarray
+    - pip
+    - wheel
+    - six 1.16.0  # [not s390x]
```

Comment:
Are six, bitarraw and thriftpy2 actually required in host

Reply:
No, no they are not. Good catch. Thank you.